hpOneView/resources/security/certificates_server.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright [2020] Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from future import standard_library
+
+standard_library.install_aliases()
+
+
+from hpOneView.resources.resource import Resource
+
+
+class CertificatesServer(Resource):
+    """
+    The Certificates Servers resource provides REST APIs for configuration of device or
+    server certificates for the appliance to establish SSL communication with other managed network entities.
+
+    Import, Update and Delete APIs are asynchronous and GET API is synchronous.
+
+    """
+    URI = '/rest/certificates'
+
+    DEFAULT_VALUES = {
+        '600': {"type": "CertificateInfoV2"},
+        '800': {"type": "CertificateInfoV2"},
+        '1000': {"type": "CertificateInfoV2"},
+        '1200': {"type": "CertificateInfoV2"}
+    }
+
+    def __init__(self, connection, data=None):
+        super(CertificatesServer, self).__init__(connection, data)
+
+    def create(self, data=None, timeout=-1, custom_headers=None):
+        """Makes a POST request to create a resource when a request body is required.
```

Comment:
docstring is too generic

Reply:
Instead of rewriting the entire method here, can you call the superclass method with the modified URI?

Reply:
Done.. Thanks.